pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1748,3 +1963,32 @@ func ExpectLaunchTemplatesCreatedWithUserData(expected string) {
 		ExpectWithOffset(2, actualUserData).To(Equal(expectedUserData))
 	})
 }
+
+func ExpectLaunchTemplatesCreatedWithUserData() []string {
```

Comment:
I'm not sure I entirely agree, it has two sets of expectations: that launch templates were created and that each of those launch templates included UserData. That UserData is the returned. I think this matches the format of most of our other expectations, e.g. `ExpectNodeCreated` where it checks that the node was created and returns the node so you can do separate checks on the values.

Reply:
Ah, maybe it was just the different way that I was interpreting this function call. What about including "Exists" at the end of it to indicate that this is an existence check rather than an equality check?

Reply:
If that was the case, then you could name the functions: "ExpectLaunchTemplatesCreatedWithUserData" and "ExpectUserDataExistsFromLaunchTemplate". Just a minor suggestion though